harled/matey
# Adding Bounce Rate Component

**Added Files:** 

- `bounce_rate_component.rb`
- `bounce_rate_component.html.erb`

**Testing Instructions**

1.  REMOVE ALL OTHER RENDERS in `index.html.erb` EXCEPT for `Matey::BounceRateComponent`
2. Follow the [Development Instructions](https://github.com/harled/matey#development) to boot up your localhost sample app
3. Open your localhost server in an **incognito tab** (Cmd + Shift + n)
4. Close the incognito tab. 
5. Refresh your localhost server in your 'normal' tab and see that the Bounce Count and Total Number of Visits has gone up.
6. Open your localhost server in an **incognito tab** again. 
7. Refresh the incognito tab and close it. 
8. Refresh your localhost server in your 'normal' tab and see that only the Total Number of Visits has gone up.
